cli/commands/migrate.ts

```diff
@@ -35,7 +35,11 @@ const allContracts = [
   'AllocationExchange',
 ]
 
-export const migrate = async (cli: CLIEnvironment, cliArgs: CLIArgs): Promise<void> => {
+export const migrate = async (
+  cli: CLIEnvironment,
+  cliArgs: CLIArgs,
+  autoMine = false,
```

Comment:
I'm leaning towards not adding anything new to the CLI unless it's a bugfix or something that's really really needed. I don't mind adding it but don't want to add stuff "just in case it's handy". Since the `autoMine` is not changing the default behavior I say we keep it as is and if needed we can add down the road.

Reply:
But it is changing the default behavior, unless I'm missing something? iirc the localhost network is set to have automine = true by default, and this change will set it to false by default.

Reply:
Are you referring to the localhost network configuration in `hardhat.config.ts`? 

If so, then yes it's true that it's set to auto-mine by default and then the `migrate` script changes that (for 1337, so including `localhost`), but that was always the case, this PR doesn't change that as the `autoMine` default is `false` in the `migrate` function.

Reply:
Uh, you're right, sorry, I missed the line setting automine to false in the diff! 🤦

Reply:
Ah! You do raise a valid point which is why are we setting auto mine to false for local networks by default.